### PR TITLE
docs(cookbox/theme-management): replaces deprecated tailwind "class" …

### PR DESCRIPTION
**Pull request by TheMcnafaha**
# What is it?
- Docs

# Description
TLDR: Fixes deprecated method.

This is a small change. At some point, the 'class' strategy was replaced by the 'selector' strategy. 

Despite the small code change, I'm marking this as a draft for one big reason:

1) I couldn't figure out the correct way to change the header/metadata of the file. 

If that gets  fixed, this should be an easy merge. Everything works as it did before (test [repo](https://github.com/TheMcnafaha/cookbook-theme-manegement) if curious).

 I plan on making a new PR after this as I want to improve the behavior of the guide. Mainly, the current guide doesn't respect the user's default theme. 

# Checklist

- [X] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [X] I performed a self-review of my own code
- [X] I made corresponding changes to the Qwik docs

### CODE REVIEW

**Comment by Shane-Donlon** [comment]
Hey there, 👋 

Only seeing this now.

The cookbook has since been updated (December) with the updated function now adhering to the 'user preference' on theme choice.

Although I did not make the change for 
darkMode: "selector" in my PR, so this update is still needed.

If you have a look at the updated cookbook and let me know if anything else needs to be changed?

If more changes are needed happy to work with you to answer the below, although clarification may be needed as to what the end result code should look like.

"I couldn't figure out the correct way to change the header/metadata of the file."

**Comment by maiieul** [comment]
closing as stale. @TheMcnafaha or @Shane-Donlon feel free to re-open a new one with the complete changes if you think it needs more, or the same and explain why nothing more is needed. Thanks for your contributions guys!
